build.gradle

```diff
@@ -16,6 +16,7 @@ ext.jerseyVersion = '2.22.2'
 ext.slf4jVersion = '1.7.16'
 ext.jacksonVersion = '2.7.4'
```

Comment:
Yeah, it seems crossdock has it's own version. It's also in use by jaeger-jaxrs2, but it seems that it's unused. 
I deleted it.